Detector/DetFCChhBaseline1/compact/FCChh_DectEmptyTracker.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Moved to Tracker directory. The point of this was to quickly be able to reduce the world size for simulations of the tracker only, since this saves a lot of cpu time. So I think it makes sense to keep the convention similar  to  `FCChh_DectEmptyMaster`.